build.sbt

```diff
@@ -14,7 +14,7 @@ libraryDependencies ++= Seq(
   "org.scalikejdbc" %% "scalikejdbc" % "2.1.2",
   "org.scalatest" %% "scalatest" % "2.1.5" % "test",
   "com.googlecode.flyway" % "flyway-core" % "2.3",
-  "me.fornever" %% "platonus" % "0.2.1",
+  "me.fornever" %% "platonus" % "0.3",
```

Comment:
I think its author have reviewed it. Do you mean you want it to be moved to codingteam foundation?